repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/image-block.ftl

```diff
@@ -0,0 +1,17 @@
+<#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"] >
+
+<#macro imageBlock imageBlock>
+    <@hst.link var="imgLink" hippobean=imageBlock.image/>
+    <figure class="nhsuk-image" style="width: 100%">
+        <img class="nhsuk-image__img" src="${imgLink}" alt="${imageBlock.image.description!}">
+        <#if imageBlock.caption?has_content>
```

Comment:
Could we simplify the `<#if>...<#elseif>` block using freemarker interpolation syntax using `then` or `string` built-in to avoid repeating `<figcaption class="nhsuk-image__caption">...</figcaption>` block ?